apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -323,6 +334,20 @@ protected static CommandLine parseArgs(String[] args) {
             .longOpt("rootPath")
             .desc("Root path of the file system")
             .build());
+    options.addOption(
+        Option.builder(null)
+            .required(false)
+            .hasArg()
+            .longOpt(OperationTasksBuilder.MAX_STRATEGIES_COUNT)
```

Comment:
Nit: Should we just call this max_tables_count?

Reply:
I don't know, I guess since we discussed that at partition granularity, there could be multiple strategies per table, the ranker could pick some partitions from one table and some from others, in the future if/when we have this granularity.